hack/lib/install.sh

```diff
@@ -33,6 +33,10 @@ function check_kind {
   if [[ $? -ne 0 ]]; then
```

Comment:
can we use one if-clause here?

Reply:
the problem is, that this the code in this if clause will not be executed. Because echo will return a success and then you test if echo is executed successfully

Reply:
This if-clause is used to test whether kind already been installed, see L32

Reply:
This if-clause is used to test whether kind already been installed

Reply:
yes I found my fault